app/models/organization.js

```diff
@@ -126,6 +126,19 @@ export default class OrganizationModel extends AgentModel {
     });
   }
 
+  get kboNumber() {
+    const identifiers = this.identifiers.content;
+
+    for (const identifier of identifiers) {
+      const structuredIdentifier = identifier.get('structuredIdentifier');
+      if (identifier.idName === 'KBO nummer') {
```

Comment:
It might also be nice to use the exported constant: https://github.com/lblod/frontend-organization-portal/blob/590bef4fdcc48eef433e566b7f7c5a59c75b517c/app/models/identifier.js#L6